server/tinylicious/src/index.ts

```diff
@@ -20,7 +20,7 @@ import { TinyliciousRunnerFactory } from "./runnerFactory";
 // a monolithic process, it still uses sockets to communicate with the historian service.  For these call,
 // keep the TCP connection open so that they can be reused
 // TODO: Set this globally since the Historian use the global Axios default instance.  Make this encapsulated.
-Axios.defaults.httpAgent = new http.Agent({ keepAlive: true });
+Axios.defaults.httpAgent = new http.Agent({ keepAlive: true, timeout: (60 * 1000) * 4 });
```

Comment:
I don't think `timeout` is not related to keepAlive, it is just a general timeout for the socket.
I beileve you will need to use [agentkeepalive](https://www.npmjs.com/package/agentkeepalive) package, which has an extra option to specify `freeSocketTimeout`.   I don't think we need to specify a number before it is already default 4000ms.